src/launch_common.py

```diff
@@ -91,8 +103,17 @@ def args_validation(args, argparser):
         logger.info("args: release_override argument does not exist.")
     else:
         release_override = args.release_override
-        if release_override not in [-11, -5, 0]:
-            argparser.error("release-override must be -11, -5 or 0. Default is 0.")
+        preserved_cycles = default_network_config_map[network]["NB_FREEZE_CYCLE"]
+        estimated_reward_override = -preserved_cycles * 2 - 1
+        frozen_reward_override = -preserved_cycles
+        if release_override not in [
+            estimated_reward_override,
+            frozen_reward_override,
+            0,
```

Comment:
Is 0 now deprecated? Shouldn't we exclude it here and make -5 as default?

Reply:
yes, 0 is now deprecated, and I agree -5 should be the default, but there are complications. I thought we could take the opportunity to migrate this parameter to the config file. We talked about this before, and you said we should have a generic way to pass parameter with either the config file or the CLI. I don't disagree, I think it requires big refactoring which is better done after this PR is merged. Let's leave the default to 0 now.